src/conduits/java/com/enderio/conduits/common/blockentity/ConduitBlockEntity.java

```diff
@@ -106,7 +109,8 @@ public void handleConnectionStateUpdate(Direction direction, ConduitType<?> cond
         updateConnectionToData(conduitType);
     }
 
-    @UseOnly(LogicalSide.SERVER)
+
+    @EnsureSide(EnsureSide.Side.CLIENT)
```

Comment:
Changed from Server to Client - is that right?

Reply:
probably not, I'll double check it